mirdata/datasets/tonas.py

```diff
@@ -0,0 +1,438 @@
+@inproceedings{tonas_music,
+    author = {Mora, Joaquin and Gómez, Francisco and Gómez, Emilia
+              and Borrego, Francisco Javier and Díaz-Báñez, José},
+    year = {2010},
+    month = {01},
+    pages = {351-356},
+    title = {Characterization and Similarity in A Cappella Flamenco Cantes.}
+}
+
+Transcriptions:
+@inproceedings{tonas_annotations,
+    author = {E. {Gómez} and J. {Bonada}},
+    journal = {Computer Music Journal},
+    title = {Towards Computer-Assisted Flamenco Transcription: An Experimental 
+           Comparison of Automatic Transcription Algorithms as Applied to A 
+           Cappella Singing},
+    year = {2013},
+    volume = {37},
+    number = {2},
+    pages = {73-90},
+    doi = {10.1162/COMJ_a_00180}}
+"""
+
+
+REMOTES = None
+
+DOWNLOAD_INFO = """
+        Unfortunately, the TONAS dataset is not available to be shared openly. However,
+        you can request access to the dataset in the following link, providing a brief
+        explanation of what your are going to use the dataset for:
+        ==> https://zenodo.org/record/1290722
+        Then, unzip the dataset and locate it to {}. If you unzip it into a different path,
+        please remember to set the right data_home when initializing the dataset.
+"""
+
+LICENSE_INFO = """
+The TONAS dataset is offered free of charge for internal non-commercial use only. You can not redistribute it nor 
+modify it. Dataset by COFLA team. Copyright © 2012 COFLA project, Universidad de Sevilla. Distribution rights granted 
+to Music Technology Group, Universitat Pompeu Fabra. All Rights Reserved.
+"""
+
+
+class NoteData(annotations.NoteData):
+    """
+    This is an extended version of standard NoteData class to support energy annotation.
+
+    Attributes:
+        intervals (np.ndarray): (n x 2) array of intervals
+            (as floats) in seconds in the form [start_time, end_time]
+            with positive time stamps and end_time >= start_time.
+        notes (np.ndarray): array of notes (as floats) in Hz
+        energies (np.ndarray): array of energies (as floats) of each note
+        confidence (np.ndarray or None): array of confidence values
+            between 0 and 1
+
+    """
+
+    def __init__(self, intervals, notes, energies, confidence=None):
+        super().__init__(intervals, notes, confidence)
+
+        annotations.validate_array_like(intervals, np.ndarray, float)
+        annotations.validate_array_like(notes, np.ndarray, float)
+        annotations.validate_array_like(energies, np.ndarray, float)
+        annotations.validate_array_like(
+            confidence, np.ndarray, float, none_allowed=True
+        )
+        annotations.validate_lengths_equal([intervals, notes, energies, confidence])
+        annotations.validate_intervals(intervals)
+        annotations.validate_confidence(confidence)
+
+        self.intervals = intervals
+        self.notes = notes
+        self.energies = energies
+        self.confidence = confidence
```

Comment:
the lines that are part of the __init__ in the base class don't need to be called here - they get called already in line 114!

This whole init can become, equivalently:
```
super().__init__(intervals, notes, confidence)
annotations.validate_array_like(energies, np.ndarray, float)
annotations.validate_lengths_equal([intervals, notes, energies, confidence])
self.energies = energies
```

Reply:
oops! You're right. Fixed now!